pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_app_zimbra_for_android_ios/guide.es-es.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Zimbra - Configurar una cuenta de correo electrónico en la aplicación móvil Zimbra"
+excerpt: "Cómo configurar una cuenta de correo electrónico de Zimbra en la aplicación móvil de Zimbra disponible en Android e iOS"
+updated: 2024-11-13
+---
+
+<style>
+.w-600 {
+  max-width:600px !important;
+}
+.h-600 {
+  max-height:600px !important;
+}
+</style>
+
+> [!warning]
+>
+> **Importante**
+>
+> El producto Zimbra es un producto en fase beta.
+>
+> Solo está disponible para las personas que hayan completado el [formulario de inscripción en la beta](https://labs.ovhcloud.com/en/zimbra-beta/).
+>
+> Algunas de las funciones o limitaciones descritas en esta guía pueden cambiar cuando el producto salga al mercado.
+
+## Objetivo
+
+Es posible configurar las cuentas de correo electrónico de Zimbra en el cliente de correo que usted utilice. para que pueda utilizar su dirección de correo desde cualquier dispositivo. La aplicación Zimbra está disponible gratuitamente en Android e iOS
+
+**Descubra cómo configurar su cuenta Zimbra en la aplicación Zimbra.**
+
+> [!warning]
+>
+> OVHcloud pone a su disposición servicios cuya configuración, gestión y responsabilidad recaen sobre usted. Por lo tanto, usted deberá asegurarse de que estos funcionen correctamente.
+>
+> Esta guía le ayudará a realizar las tareas más habituales. No obstante, si necesita ayuda, le recomendamos que contacte con un [partner especializado](https://marketplace.ovhcloud.com/c/support-collaboration) o con el editor del servicio. Nosotros no podremos asistirle. Para más información, consulte el apartado «Más información» de esta guía.
```

Comment:
https://marketplace.ovhcloud.com/c/support-collaboration => available for all subs ?

Reply:
why don't you use `/links/partner` instead ?